### DISCUSSION TITLE
handle Gmail's `+` behaviour

### DISCUSSION THREAD

**Discussion by arpitdalal** [epicweb-dev/epic-stack#536] [Q&A]
Gmail treats ` example@gmail.com` and `example+app@gmail.com` as the same address, but the epic stack allows users to sign up using these `+` variations, meaning users can create as many accounts using this technique and exploit a freemium model app.
How would you approach to solve this?
Here's what I think I would do: create a new column in DB to store the original value with `+` which can be used in communication if someone actually uses this technique to filter their inbox; and remove the `+` part of the email and store it with a unique constraint to check on this column with new signups

**Comment by kentcdodds** [comment]
This is something that I would build into the epic stack. Sometimes this is a desirable feature. But if you wanted to handle this yourself, then I would recommend adding validation that prevents using + in the email before the @ sign. 

Because this is not something we're going to add to the epic stack, I'm going to close this. But feel free to create an example if you're interested!
